src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -390,380 +394,382 @@ export class BaseEditCourseForm extends React.Component {
               className="course-image"
               disabled={disabled}
             />
-            <hr />
```

Comment:
The rest of this file is just wrapping a huge section in `{showMarketingFields && <React.Fragment>` and indenting.